govulncheck: fix extract mode description (#23698)
tldr-pages/tldr: pages/common/govulncheck.md

`govulncheck -mode binary {{path/to/binary}}`

> - Extract the information needed to analyze a Go binary:

Reviewer: ```suggestion
- Extract build information and build dependencies from a go binary:
```

Reviewer: I promise i definitely wanted to say ___build dependencies___ and not [build information](https://github.com/tldr-pages/tldr/pull/23598#pullrequestreview-4996199963) again :)
and I have no idea I managed to have typed that up, must've been half asleep.

Author: Applied in signed commit 615674633. I kept the repository spelling for Go and re-ran page-specific markdownlint/tldr-lint plus the complete repository test script; all passed. Thanks for clarifying the intended build-dependencies wording.
